src/app/components/Curation/CurationGrid/index.tsx

```diff
@@ -1,20 +1,50 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/react';
+import moment from 'moment';
 import styles from './index.styles';
 import CurationPromo from '../CurationPromo';
 import { CurationGridProps } from '../types';
+import { Summary } from '../../../models/types/curationData';
 
 const CurationGrid = ({
   summaries,
   isFirstCuration,
   headingLevel,
+  eventTrackingData,
 }: CurationGridProps) => {
   const hasMultiplePromos = summaries.length > 1;
   const firstPromo = summaries[0];
 
   if (summaries.length === 0) {
     return null;
   }
+
+  const buildPromoEventTrackingData = (
+    promo: Summary,
+    i: number,
+    { readTime }: { readTime?: number } = {},
+  ) => ({
+    itemTracker: {
+      type: 'simple-curation-grid-promo',
+      text: promo.title,
+      position: i + 1,
+      resourceId: promo.id,
+      ...(promo.type && { mediaType: promo.type }),
+      ...(promo.duration && {
+        duration: moment.duration(promo.duration, 'seconds').asMilliseconds(),
+      }),
+      ...(readTime && {
+        label:
+          readTime === 1
+            ? `Read time: ${readTime} minute`
+            : `Read time: ${readTime} minutes`,
+        // duration: readTime * 60000,
```

Comment:
Main addition here. Originally I just copied the same usage for `duration` from the articles experiment, but it seems like it might be used here as well. Unsure if there is actually a case where this will clash and overwrite some data intended for elsewhere, but worth mentioning